pkg/virt-handler/domain.go

```diff
@@ -36,11 +38,10 @@ type DomainDispatch struct {
 }
 
 func (d *DomainDispatch) Execute(indexer cache.Store, queue workqueue.RateLimitingInterface, key interface{}) {
-	obj, exists, err := indexer.GetByKey(key.(string))
-	if err != nil {
-		queue.AddRateLimited(key)
-		return
-	}
+	obj, exists, _ := indexer.GetByKey(key.(string))
+	// GetByKey can never return anything but nil for the error
```

Comment:
@admiyo , you could use your own store implementation (A store is an interface) which can throw an error, if that is your line of agrumentation. Every code in kubernetes itself is also taking  this error into account.